modules.yml

```diff
@@ -1020,3 +1020,13 @@
   path: recon/netblocks-hosts/reverse_resolve
   required_keys: []
   version: '1.0'
+- author: Ryan Hays
+  dependencies: []
+  description: Uses the BinaryEdge API to discover subdomains.
+  files: []
+  last_updated: '2019-05-28'
+  name: BinaryEdge.io DNS lookup
+  path: dev/binaryedge_hosts
```

Comment:
I bet this was the path in your dev environment, but please update it to where the module should show up in the recon-ng module structure.